<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Research | Semmelweis University Computational Biology Group</title>
<meta name="generator" content="Jekyll v3.8.3" />
<meta property="og:title" content="Research" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Semmelweis University Computational Biology Group at the 1st Department of Pathology and Experimental Cancer Research. H-1085 Budapest, Üllői út 26. Hungary" />
<meta property="og:description" content="Semmelweis University Computational Biology Group at the 1st Department of Pathology and Experimental Cancer Research. H-1085 Budapest, Üllői út 26. Hungary" />
<link rel="canonical" href="http://localhost:4000/research/" />
<meta property="og:url" content="http://localhost:4000/research/" />
<meta property="og:site_name" content="Semmelweis University Computational Biology Group" />
<script type="application/ld+json">
{"url":"http://localhost:4000/research/","headline":"Research","description":"Semmelweis University Computational Biology Group at the 1st Department of Pathology and Experimental Cancer Research. H-1085 Budapest, Üllői út 26. Hungary","@type":"WebPage","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<link rel="stylesheet" href="/assets/main.css"><link type="application/atom+xml" rel="alternate" href="http://localhost:4000/feed.xml" title="Semmelweis University Computational Biology Group" /></head>
<body><header class="site-header" role="banner">

  <div class="wrapper"><a class="site-title" rel="author" href="/">Semmelweis University Computational Biology Group</a><nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.032C17.335,0,18,0.665,18,1.484L18,1.484z M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.032C17.335,6.031,18,6.696,18,7.516L18,7.516z M18,13.516C18,14.335,17.335,15,16.516,15H1.484 C0.665,15,0,14.335,0,13.516l0,0c0-0.82,0.665-1.483,1.484-1.483h15.032C17.335,12.031,18,12.695,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger"><a class="page-link" href="/news/">Group News</a><a class="page-link" href="/research/">Research</a><a class="page-link" href="/people/">People</a><a class="page-link" href="/publications/">Publications</a><a class="page-link" href="/positions/">Positions</a><a class="page-link" href="/resources/">Resources</a></div>
      </nav></div>
</header>
<main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post">

  <div class="post-content">
    <p>We are interested in the following research areas connected to the molecular
biology of cancer, and more broadly, to human development and disease. We aim to
develop novel computational tools, and provide biological insight using
large-scale datasets from high-throughput experiments.</p>

<h3>The role of non-coding and synonymous genetic variants in shaping the
transcriptome</h3>

<p>In cancer and other genetic diseases, mutations frequently occur in the
non-coding functional regions of the genome. These mutations affect
transcriptional enhancers, promoters or the various elements regulating
splicing. Non-pathogenic population variants are also frequently found in these
regions. However, the exact role of these mutations and variants in regulating
protein – DNA and protein – RNA interactions, RNA expression, splicing, folding,
stability, localization and translation is unkown in many cases.</p>

<p>Variants might change the DNA shape and local structure of promoter and enhancer
regions, leading to changes in transcription factor binding strength and gene
expression. However, the effect of single nucleotide mutations and small indels
on structural features and how they influence cancer susceptibility, development
and progression through changing transcription factor binding is mostly
uncharted territory. On the other hand, synonymous variants in coding regions
might alter protein – RNA interactions, the addition of RNA post-transcriptional
modifications, and RNA secondary structure. Most of these variants probably have
a small effect, but contribute to a large number of phenotypes, requiring
systems biology approaches to analyse their role.</p>

<h3>The role of RNA binding proteins in human disease</h3>

<p>More than 1500 genes encoding RNA binding proteins (RBPs) exist in the human
genome. They regulate the splicing, polyadenylation, export, localization, decay
and translation of coding and non-coding RNAs. However, a large number of these
RBPs are still uncharacterized, and their exact role is unknown in shaping the
transcriptome during development and disease. An outstanding problem is that
RBPs have less sequence specificity compared to transcription factors, and
probably work in a highly overlapping and combinatorial way. Several RBP
mutations contribute to disease, including neurodegeneration and cancer.</p>

<p>Amyotrophic lateral sclerosis (ALS), an untreatable neurodegenerative disease
harbors frequent mutations in RNA binding protein genes, but it is not known how
the different mutations lead to the disease and what are the final downstream
targets. An outstanding question is whether different RBP mutations cause
similar changes in RNA processing and stability or influence distinct genes.
Additionally, the penetrance of these mutations is often below 50% and there is
evidence for polygenic inheritance. Haematologic cancers, including myeloid and
lymphocytic leukemias or the myelodysplastic syndrome also contain
characteristic RNA binding protein mutations. The effect of these mutations on
RNA levels and processing are not well understood, although the exact location
of the mutations on the protein can differentiate even histological subtypes of
the disease. Comprehensive reanalysis of existing literature data (RNA-seq,
CLIP-seq, RIP-seq, etc) will shed light on some of these questions.</p>

<h3>Transcriptome deregulation, gene essentiality and redundancy in cancer</h3>

<p>Cancer cells acquire mutations, leading to a large number of splicing, gene
expression and biological pathway changes. Current large-scale projects are
characterizing these changes in different cancer types. Some pediatric cancers
have no somatic mutations, while certain colorectal and endometrial cancers are
hypermutated and have more than 100 mutations per megabase. However, the
connection between the amount and type of mutations, the extent of transcriptome
deregulation and the development of cancer is not well understood.</p>

<p>The similarity or difference in transcriptome deregulation between different
cancer types is largely uncharacterized. Clarifying this question is hindered by
the fact that most investigations and methods describe shifts in mean expression
values associated to mutations. Nonetheless, changes in expression variability
on the splicing variant, gene or pathway level might be just as important and
characteristic for different cancers and describing them will be crucial to
understand cancer development and progression in depth. Cancer can also be
understood as a minimal organism with a minimal gene regulatory network, where a
large number of genes can be mutated or deregulated without functional
consequences. Using cancer as a data-rich “model animal” we can investigate
generally relevant biological questions. As an example, using hypermutated tumor
it would be possible to describe essential and non-essential genes or essential
molecular functions for different cellular or tissue contexts. Complementing
this data with other public datasets, we can also identify functionally
redundant gene pairs or larger groups. These might include transcription factor
or splicing factors groups and homologous or non-homologous isoenzymes.  The
detailed characterization of essential and redundant genes or molecular
functions will help in understanding complex polygenic inheritance.</p>

  </div>

</article>

      </div>
    </main><footer class="site-footer h-card">
  <data class="u-url" href="/"></data>

  <div class="wrapper">

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li class="p-name">Semmelweis University Computational Biology Group</li><li><a class="u-email" href="mailto:endre.sebestyen@gmail.com">endre.sebestyen@gmail.com</a></li></ul>
      </div>

      <div class="footer-col footer-col-2"><ul class="social-media-list"><li><a href="https://github.com/SU-CompBio"><svg class="svg-icon"><use xlink:href="/assets/minima-social-icons.svg#github"></use></svg> <span class="username">SU-CompBio</span></a></li><li><a href="https://www.twitter.com/SU_CompBio"><svg class="svg-icon"><use xlink:href="/assets/minima-social-icons.svg#twitter"></use></svg> <span class="username">SU_CompBio</span></a></li></ul>
</div>

      <div class="footer-col footer-col-3">
        <p>Semmelweis University Computational Biology Group at the 1st Department of Pathology and Experimental Cancer Research. H-1085 Budapest, Üllői út 26. Hungary</p>
      </div>


    </div>

  </div>

</footer>
</body>

</html>
